internal/app/cache/cache.go

```diff
@@ -0,0 +1,31 @@
+package cache
+
+import (
+	envoy_service_discovery_v3 "github.com/envoyproxy/go-control-plane/envoy/service/discovery/v3"
```

Comment:
may be use v2 for now? Once we have the v2/v3 story figured in g-c-p we can introduce the same technique here.

Reply:
I'm fine with using v2 here, but the spec definitions all use v3 types. Have we decided that the MVP of ids-relay will use v2, unless we do the v2/v3 conversion before we finish the MVP?

Reply:
I think we should be making sure xds-relay can talk to g-c-p and that means going with v2 as of now. we are going to support both versions. for now the next work stream to connect to control plane might get blocked because of using v3.